site/shared/src/main/scala/shared/User.scala

```diff
@@ -19,8 +19,8 @@ case class UserProgress(
   sectionName:  String,
   method:       String,
   version:      Int,
-  exerciseType: String         = "Other",
-  args:         Option[String],
+  exerciseType: String               = "Other",
+  args:         Option[List[String]],
```

Comment:
I'm not convinced that this double nesting is required and I think it can be considered technical debt with an appropriate issue logged if this isn't solved in this PR.